### DISCUSSION TITLE
Can someone tell me how much storage space would I need to install the desktop App

### DISCUSSION THREAD

**Discussion by Outshine8** [pieces-app/support#422] [Q&A] [answered]
I want to install the Desktop App for windows but I don't have too much storage space to spare

**Comment by rosie-at-pieces** [comment]
@nathan-courtney-pieces

**Comment by nathan-courtney-pieces** [comment, marked as answer]
@KlyntarX69 

Both Pieces OS and Pieces for Developers will take roughly 1GB of space on your machine after installed. 

Depending on how you use it, your User Data Directory could be around 100MB if you store a lot of snippets and interreact with it a lot. 

If you download our local models through the application, it can vary but most of them are over 3GB, typically around 5GB

**Reply by Outshine8** [reply]
Thx
